### PR TITLE
Add 2021Q3 entries from doceng

### PR DESCRIPTION
**Pull request by dbaio**

### CODE REVIEW

File: 2021q3/doceng.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+////
+Quarter:	3rd quarter of 2021
+Prepared by:	dbaio
+Reviewed by:	gjb, carlavilla
+Last edit:	$Date: 2021-10-03 05:34:14 -0300 (Sun, 03 Oct 2021) $
+Version:	$Id: doceng-2021-3rd-quarter-status-report.adoc 185 2021-10-03 08:34:14Z carlavilla $
+////
+
```
Comment (pauamma) [inline 2021q3/doceng.adoc:8]: Assuming the block above is part of doceng workflow (and therefore way above my pay grade) and has a good reason to be there.

File: 2021q3/doceng.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org>
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project, more infomation on link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
```
Comment (pauamma) [inline 2021q3/doceng.adoc:17]: s/Project, more infomation on/Project; for more information, see/ (mind the missing "r" in "information" even if you don't agree with the whole suggestion)

**Review by pauamma — CHANGES_REQUESTED** [review]
Other than that; LGTM.

File: 2021q3/doceng.adoc (hunk 1 above)
Comment (dbaio) [reply 2021q3/doceng.adoc:8]: This is just a comment.
if necessary, you can remove it.

File: 2021q3/doceng.adoc (hunk 2 above)
Comment (dbaio) [reply 2021q3/doceng.adoc]: Thanks. I've just pushed a fix as suggested.

**Review by pauamma — APPROVED** [review]
LGTM.

File: 2021q3/doceng.adoc (hunk 1 above)
Comment (pauamma) [reply 2021q3/doceng.adoc:8]: I'll let @debdrup decide.

Comment (debdrup) [reply 2021q3/doceng.adoc:8]: The metadata wasn't present in the doceng entry in the last status report, and nor do any of the other entries include similar metadata - so I don't see a reason to include it.

**Pull request merged**